backend/src/util/VerifyFIN.test.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
```

Comment:
Why `eslint-disable`?

Reply:
Thought it was a workaround for using tests files but I managed to find another method which adds jest env to eslint, makes much more sense I believe. Thanks for pointing this out!